src/app/resources/resources.component.ts

```diff
@@ -121,7 +129,30 @@ export class ResourcesComponent implements OnInit, AfterViewInit {
       this.couchService
         .get('resources/_all_docs?include_docs=true')
         .subscribe(data => {
-          this.resources.data = data.rows.map(res => res.doc);
+          this.resources.data = data.rows.map(res => {
```

Comment:
Please use the merging functions from #270.  We should avoid nesting observables and should never repeat calls to the database to fetch each resources' rating one by one.